docroot/themes/humsci/humsci_basic/patterns/spotlight/spotlight.ui_patterns.yml

```diff
@@ -0,0 +1,24 @@
+spotlight:
```

Comment:
Can you please add this to the old theme. it doesn't need to have the styling, but the pattern has to be available in each theme that is available.